components/odh-notebook-controller/controllers/notebook_runtime_test.go

```diff
@@ -34,107 +35,196 @@ var _ = Describe("When Creating a notebook should mount the configMap", func() {
 			name              string
 			notebook          *nbv1.Notebook
 			notebookName      string
-			ConfigMap         *unstructured.Unstructured
+			ConfigMap         *corev1.ConfigMap
+			imageStream       *imagev1.ImageStream
```

Comment:
maybe a comment that either imagestream or configmap is to be provided? didn't you say you want to remove specifying configmap?